json-server/custom-body-mapper.js

```diff
@@ -10,5 +10,17 @@ module.exports = (req, res, next) => {
             req.body = rest;
         }
     }
+
+    if (req.method === "GET" && req.url.includes("/users")) {
+        if (getUser(req) === "finance-user") {
+            req.url = `/financeUser`;
```

Comment:
:shrug: I have no idea if this is the best way but it looked like the only way to add the role to the response would be to use `json-server` as a module, which I didn't fancy doing